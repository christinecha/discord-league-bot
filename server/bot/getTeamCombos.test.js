const { getTeamCombos } = require('./getTeamCombos')

test('get 4s combos', async (done) => {
  const combos = getTeamCombos(4)
  expect(combos).toMatchSnapshot()
  done()
})

test('get 3s combos', async (done) => {
  const combos = getTeamCombos(3)
  expect(combos).toMatchSnapshot()
  done()
})

test('get 2s combos', async (done) => {
  const combos = getTeamCombos(2)
  expect(combos).toMatchSnapshot()
  done()
})

test('get 1s combos', async (done) => {
  const combos = getTeamCombos(1)
  expect(combos).toMatchSnapshot()
  done()
})
