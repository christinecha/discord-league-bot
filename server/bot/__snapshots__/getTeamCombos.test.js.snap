// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get 1s combos 1`] = `
Array [
  Array [
    1,
    2,
  ],
]
`;

exports[`get 2s combos 1`] = `
Array [
  Array [
    1,
    1,
    2,
    2,
  ],
  Array [
    1,
    2,
    1,
    2,
  ],
  Array [
    1,
    2,
    2,
    1,
  ],
]
`;

exports[`get 3s combos 1`] = `
Array [
  Array [
    1,
    1,
    1,
    2,
    2,
    2,
  ],
  Array [
    1,
    1,
    2,
    1,
    2,
    2,
  ],
  Array [
    1,
    1,
    2,
    2,
    1,
    2,
  ],
  Array [
    1,
    1,
    2,
    2,
    2,
    1,
  ],
  Array [
    1,
    2,
    1,
    1,
    2,
    2,
  ],
  Array [
    1,
    2,
    1,
    2,
    1,
    2,
  ],
  Array [
    1,
    2,
    1,
    2,
    2,
    1,
  ],
  Array [
    1,
    2,
    2,
    1,
    1,
    2,
  ],
  Array [
    1,
    2,
    2,
    1,
    2,
    1,
  ],
  Array [
    1,
    2,
    2,
    2,
    1,
    1,
  ],
]
`;

exports[`get 4s combos 1`] = `
Array [
  Array [
    1,
    1,
    1,
    1,
    2,
    2,
    2,
    2,
  ],
  Array [
    1,
    1,
    1,
    2,
    1,
    2,
    2,
    2,
  ],
  Array [
    1,
    1,
    1,
    2,
    2,
    1,
    2,
    2,
  ],
  Array [
    1,
    1,
    1,
    2,
    2,
    2,
    1,
    2,
  ],
  Array [
    1,
    1,
    1,
    2,
    2,
    2,
    2,
    1,
  ],
  Array [
    1,
    1,
    2,
    1,
    1,
    2,
    2,
    2,
  ],
  Array [
    1,
    1,
    2,
    1,
    2,
    1,
    2,
    2,
  ],
  Array [
    1,
    1,
    2,
    1,
    2,
    2,
    1,
    2,
  ],
  Array [
    1,
    1,
    2,
    1,
    2,
    2,
    2,
    1,
  ],
  Array [
    1,
    1,
    2,
    2,
    1,
    1,
    2,
    2,
  ],
  Array [
    1,
    1,
    2,
    2,
    1,
    2,
    1,
    2,
  ],
  Array [
    1,
    1,
    2,
    2,
    1,
    2,
    2,
    1,
  ],
  Array [
    1,
    1,
    2,
    2,
    2,
    1,
    1,
    2,
  ],
  Array [
    1,
    1,
    2,
    2,
    2,
    1,
    2,
    1,
  ],
  Array [
    1,
    1,
    2,
    2,
    2,
    2,
    1,
    1,
  ],
  Array [
    1,
    2,
    1,
    1,
    1,
    2,
    2,
    2,
  ],
  Array [
    1,
    2,
    1,
    1,
    2,
    1,
    2,
    2,
  ],
  Array [
    1,
    2,
    1,
    1,
    2,
    2,
    1,
    2,
  ],
  Array [
    1,
    2,
    1,
    1,
    2,
    2,
    2,
    1,
  ],
  Array [
    1,
    2,
    1,
    2,
    1,
    1,
    2,
    2,
  ],
  Array [
    1,
    2,
    1,
    2,
    1,
    2,
    1,
    2,
  ],
  Array [
    1,
    2,
    1,
    2,
    1,
    2,
    2,
    1,
  ],
  Array [
    1,
    2,
    1,
    2,
    2,
    1,
    1,
    2,
  ],
  Array [
    1,
    2,
    1,
    2,
    2,
    1,
    2,
    1,
  ],
  Array [
    1,
    2,
    1,
    2,
    2,
    2,
    1,
    1,
  ],
  Array [
    1,
    2,
    2,
    1,
    1,
    1,
    2,
    2,
  ],
  Array [
    1,
    2,
    2,
    1,
    1,
    2,
    1,
    2,
  ],
  Array [
    1,
    2,
    2,
    1,
    1,
    2,
    2,
    1,
  ],
  Array [
    1,
    2,
    2,
    1,
    2,
    1,
    1,
    2,
  ],
  Array [
    1,
    2,
    2,
    1,
    2,
    1,
    2,
    1,
  ],
  Array [
    1,
    2,
    2,
    1,
    2,
    2,
    1,
    1,
  ],
  Array [
    1,
    2,
    2,
    2,
    1,
    1,
    1,
    2,
  ],
  Array [
    1,
    2,
    2,
    2,
    1,
    1,
    2,
    1,
  ],
  Array [
    1,
    2,
    2,
    2,
    1,
    2,
    1,
    1,
  ],
  Array [
    1,
    2,
    2,
    2,
    2,
    1,
    1,
    1,
  ],
]
`;
